src/agent/src/test_utils.rs

```diff
@@ -85,4 +85,15 @@ pub mod test_utils {
             }
         };
     }
+
+    #[macro_export]
+    macro_rules! skip_loop_by_user {
```

Comment:
Just saying, I am not a big fan of how the macro was names, but I don't have a much better suggestion.